src/main/java/org/apache/pirk/query/wideskies/QueryInfo.java

```diff
@@ -96,6 +98,33 @@ public QueryInfo(UUID identifierInput, int numSelectorsInput, int hashBitSizeInp
     printQueryInfo();
   }
 
+  public QueryInfo(Map queryInfoMap)
```

Comment:
This sounds like a good idea.  If I don't do it in this commit, it will likely go in later.